python/ml4ir/base/data/tfrecord_reader.py

```diff
@@ -261,13 +261,8 @@ def _parse_fn(proto):
             # Extract the label feature to return separately
             labels = features_dict.pop(self.feature_config.get_label(key="name"))
 
-            if self.feature_config.aux_label:
-                aux_labels = features_dict.get(self.feature_config.get_aux_label(key="node_name"))
-                # return X and y and y_aux which can be used with fit(), predict() and evaluate()
-                return features_dict, {self.output_name: labels, self.aux_output_name: aux_labels}
-            else:
-                # return X and y which can be used with fit(), predict() and evaluate()
-                return features_dict, labels
+            # return X and y which can be used with fit(), predict() and evaluate()
+            return features_dict, labels
```

Comment:
Not sure what's going on with auxiliary loss here, are they entirely removed, renamed, or refactored?